detox/e2e/test/autocomplete/at_mention.e2e.js

```diff
@@ -0,0 +1,148 @@
+    });
+
+    it('MM-Txxxx should suggest user based on lowercase last name', async () => {
+        // # Type "@" to activate at mention autocomplete
+        await expect(element(by.id('autocomplete.at_mention.list'))).not.toExist();
+        await postInput.typeText('@');
+        await expect(element(by.id('autocomplete.at_mention.list'))).toExist();
+
+        // # Type lowercase last name
+        await postInput.typeText(user.last_name.toLowerCase());
+
+        // * Expect at mention autocomplete to contain associated user suggestion
+        await expect(element(by.id(`autocomplete.at_mention.item.${user.id}`))).toExist();
+    });
+
+    it('MM-Txxxx should suggest user based on full name with space', async () => {
+        // # Type "@" to activate at mention autocomplete
+        await expect(element(by.id('autocomplete.at_mention.list'))).not.toExist();
+        await postInput.typeText('@');
+        await expect(element(by.id('autocomplete.at_mention.list'))).toExist();
+
+        // # Type full name
+        await postInput.typeText(`${user.first_name} ${user.last_name}`);
+
+        // * Expect at mention autocomplete to contain associated user suggestion
+        await expect(element(by.id(`autocomplete.at_mention.item.${user.id}`))).toExist();
+    });
+
+    it('MM-Txxxx should suggest user based on partial full name with space', async () => {
+        // # Type "@" to activate at mention autocomplete
+        await expect(element(by.id('autocomplete.at_mention.list'))).not.toExist();
+        await postInput.typeText('@');
+        await expect(element(by.id('autocomplete.at_mention.list'))).toExist();
+
+        // # Type partial full name
+        await postInput.typeText(`${user.first_name} ${user.last_name.substring(0, user.last_name.length - 6)}`);
+
+        // * Expect at mention autocomplete to contain associated user suggestion
+        await expect(element(by.id(`autocomplete.at_mention.item.${user.id}`))).toExist();
+    });
+
+    it('MM-Txxxx should stop suggesting user after full name with trailing space', async () => {
+        // # Type "@" to activate at mention autocomplete
+        await expect(element(by.id('autocomplete.at_mention.list'))).not.toExist();
+        await postInput.typeText('@');
+        await expect(element(by.id('autocomplete.at_mention.list'))).toExist();
+
+        // # Type full name
+        await postInput.typeText(`${user.first_name} ${user.last_name}`);
+
+        // * Expect at mention autocomplete to contain associated user suggestion
+        await expect(element(by.id(`autocomplete.at_mention.item.${user.id}`))).toExist();
+
+        // # Type trailing space
+        await postInput.typeText(' ');
+
+        // * Expect at mention autocomplete not to contain associated user suggestion
+        await expect(element(by.id(`autocomplete.at_mention.item.${user.id}`))).not.toExist();
+    });
+});
```

Comment:
Please add a 10th case `MM-T3409_10 should stop suggesting user when keyword is not associated with any user`

Reply:
Done - hope it's alright!